wobgob/azerothcore-wotlk
# Add Call to Arms Switch module

## Changes Proposed:
-  Add module-cta-switch

## Issues Addressed:

- Call to arms on battlegrounds that are not currently available.

## How to Test the Changes:


1. Login and check the battleground interface panel
2. Either WSG or AB should be displayed as call to arms

Required Edits:
- Battleground.RewardWinnerArenaFirst = 25 to attleground.RewardWinnerArenaFirst = 0 in worldserver.conf
